### DISCUSSION TITLE
Wo landet das Ergebnis von '{"mode": "read-raw", "data":[271,274]}' wenn auf 'open3e/cmnd' geschickt?

### DISCUSSION THREAD

**Discussion by AnHardt** [open3e/open3e#82] [Q&A]
Oder genauer: Wie komme ich vom Home Assistant aus an das Ergebnis?

Was wir zur Zeit beim _blind_ Schreiben auf komplexe Dids machen, ist ein wenig riskant. Besser wäre: auslesen, maskieren, einfügen, schreiben.

**Comment by MyHomeMyData** [comment]
Das Ergebnis wird mit dem MQTT-Topic versendet, das Du in der Kommandozeile unter der Option -m angegeben hast. Also mit `-m localhost:1883:open3e` wird alles mit dem Topic `open3e/` versendet. Im MQTT-Broker also `open3e/#` abonnieren.
